### PR TITLE
User WS name instead WS ID for root projects folder

### PR DESCRIPTION
**Pull request by vparfonov**
@skabashnyuk @garagatyi

### CODE REVIEW

File: plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java [hunk 1]
```diff
@@ -91,7 +108,18 @@ public String getPath(@Assisted("workspace") String workspaceId) throws IOExcept
...
             return hostProjectsFolder;
         } else {
-            Path folder = Paths.get(workspacesMountPoint).resolve(workspaceId);
+            String wsName;
+            try {
+                WorkspaceManager workspaceManager = this.workspaceManager.get();
+                final UsersWorkspace workspace = workspaceManager.getWorkspace(workspaceId);
+                wsName = workspace.getName();
+            } catch (BadRequestException | NotFoundException | ServerException e)
+            {
```
Comment (skabashnyuk) [inline plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java]: wrong formatting

File: plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java [hunk 2]
```diff
@@ -45,30 +53,39 @@
...
      * For example:
-     * if you set "host.workspaces.root" to the /home/user/che/workspaces after creating new workspace will be created new folder
-     * /home/user/che/workspaces/{workspaceId} and it will be mount to the  dev-machine to "che.machine.projects.internal.storage"
+     * if you set "che.user.workspaces.storage" to the /home/user/che/workspaces after creating new workspace will be created new folder
+     * /home/user/che/workspaces/{workspaceName} and it will be mount to the  dev-machine to "che.machine.projects.internal.storage"
      */
     final String workspacesMountPoint;
 
+    private final Provider<WorkspaceManager> workspaceManager;
+    private static final Logger LOG = LoggerFactory.getLogger(LocalWorkspaceFolderPathProvider.class);
```
Comment (sleshchenko) [inline plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java]: Please place static field upper than not static fields

File: plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java [hunk 3]
```diff
@@ -45,30 +53,39 @@
      * Inside machine it will point to the directory described by @see che.machine.projects.internal.storage.
      *
      * For example:
-     * if you set "host.workspaces.root" to the /home/user/che/workspaces after creating new workspace will be created new folder
-     * /home/user/che/workspaces/{workspaceId} and it will be mount to the  dev-machine to "che.machine.projects.internal.storage"
+     * if you set "che.user.workspaces.storage" to the /home/user/che/workspaces after creating new workspace will be created new folder
+     * /home/user/che/workspaces/{workspaceName} and it will be mount to the  dev-machine to "che.machine.projects.internal.storage"
      */
     final String workspacesMountPoint;
```
Comment (sleshchenko) [inline plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java]: Can we do this field private?

File: plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java (hunk 1 above)
Comment (vparfonov) [inline plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java]: fixed

File: plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java (hunk 2 above)
Comment (vparfonov) [inline plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java]: fixed

File: plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java (hunk 3 above)
Comment (vparfonov) [inline plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java]: easy

**Comment by garagatyi** [comment]
ok

**Pull request merged**